### PR TITLE
chore(svg-iconography): add icon

### PR DESCRIPTION
**Pull request by jefflembeck**
Add comment box icon for collaboration and security pane on new homepage

### CODE REVIEW

File: src/pivotal-ui/components/svg-iconography/comment-box.svg [hunk 1]
```diff
@@ -0,0 +1,9 @@
+<svg viewBox="0 0 24 21" xmlns="http://www.w3.org/2000/svg">
+  <title>
```
Comment (aredridel) [inline src/pivotal-ui/components/svg-iconography/comment-box.svg]: Do we need the title?

File: src/pivotal-ui/components/svg-iconography/comment-box.svg [hunk 2]
```diff
@@ -0,0 +1,9 @@
...
+    Group 44
+  </title>
+  <g fill-rule="evenodd">
```
Comment (aredridel) [inline src/pivotal-ui/components/svg-iconography/comment-box.svg]: Does the fill-rule affect the rendering?

File: src/pivotal-ui/components/svg-iconography/comment-box.svg (hunk 1 above)
Comment (jefflembeck) [inline src/pivotal-ui/components/svg-iconography/comment-box.svg]: Likely no. I did some copy-pasta from the asset provided and some hyper quick removal of attrs that were blocking me (not looking at others). I can go through and be more thorough on that.

**Comment by jefflembeck** [comment]
Fixed issues mentioned.

**Pull request merged**